### ISSUE TITLE
Windows app continues to run on the background when closed

### ISSUE THREAD

**Issue by CVasilakis** [danjulio/tCam#14]
I noticed the issue when i tried to move the folder which contains the tCam windows app. An error appeared saying that i can't move the folder because the app was still running. Although i had closed the app. Then i opened the task manager to check the running apps. There i saw that each time i opened the tCam app a new task was spawned, which did not clear after closing the app. Below i attach a sceenshot from the task manager:
![Screenshot 2024-08-15 150726](https://github.com/user-attachments/assets/bdcdfcbc-9633-44ae-b54c-496919cc5c8a)
As you can see there are 3 background processes for the tCam (each of which consume considerable amount of RAM - even closed). This is exactly the number that i had opened and closed the tCam app (3 times). Please note that the tCam app was closed all this times (not minimized) and that there is no instance of the app in the taskbar tray.

**Comment by danjulio** [comment]
Oh shoot.  I think I know what’s going on.  Did you close the app using the window close decoration or from the pull-down menu to quit it?  If you closed it from the window itself then I think I must not be catching that as a reason to exit it.

**Comment by CVasilakis** [comment]
> Oh shoot. I think I know what’s going on. Did you close the app using the window close decoration or from the pull-down menu to quit it? If you closed it from the window itself then I think I must not be catching that as a reason to exit it.

You are right. I closed the program using the window close decoration. I just tested what you wrote. I confirm that if i use the "exit" button from the pull-down menu, then there is no such problem. Though, i think that both actions (closing from the decoration or the pull-down manu) should have the same behavior, otherwise the user should be questioned whether he wants the app to run on the background when the "X" decoration button is pressed.
